Plugins/lib/windows/arm64/gilzoide-sqlite-net.dll.meta

```diff
@@ -21,8 +21,10 @@ PluginImporter:
         Exclude Linux64: 0
         Exclude OSXUniversal: 0
         Exclude WebGL: 1
-        Exclude Win: 1
+        Exclude Win: 0
```

Comment:
As far as I know, "Win" platform is the x86 version of Windows.
I don't think this is supposed to be enabled 🤔